docs/page-objects/rules.rst

```diff
@@ -1,782 +1,166 @@
-.. _rules-intro:
+.. _rules:
 
-Apply rules
-===========
+=====
+Rules
+=====
 
-Basic Usage
------------
+Rules are :class:`~.ApplyRule` objects that tell web-poet which :ref:`page
+object class <page-object-classes>` to use for a given combination of input URL
+and :ref:`item type <item-types>`.
 
-@handle_urls
-~~~~~~~~~~~~
+Defining rules
+==============
 
-web-poet provides a :func:`~.handle_urls` decorator, which allows to
-declare how a page object can be used (applied):
+The :func:`~.handle_urls` decorator is the simplest way to define a rule for
+a page object. For example:
 
-* for which websites / URL patterns it works,
-* which data type (item classes) it can return,
-* which page objects it can replace (override; more on this later).
+.. _handle_url_example:
 
 .. code-block:: python
 
     from web_poet import ItemPage, handle_urls
+
     from my_items import MyItem
 
     @handle_urls("example.com")
     class MyPage(ItemPage[MyItem]):
-        # ...
-
-
-``handle_urls("example.com")`` can serve as documentation, but it also enables
-getting the information about page objects programmatically.
```

Comment:
This might not be the best way to explain it, but I think the new docs lack some information about why would anyone want to use rules.

Reply:
I have restored these 2 advantages in the page introduction, and covered how rules are needed to get item objects directly or to implement generic pave object classes with website-specific overrides.